resources/js/Components/ComponentsPraktikans/PraktikanNav.jsx

```diff
@@ -23,28 +23,50 @@ function PraktikanNav() {
             setIsCollapsed(!isCollapsed);
         }
     };
+    const [isOpen, setIsOpen] = useState(false);
+    const genericHamburgerLine = `h-1 w-6 my-1 rounded-full bg-black transition ease transform duration-300`;
 
     return (
         <nav className="h-screen flex items-center">
             <div
-                className={` flex flex-col justify-between ${
+                className={` flex flex-col h-[79vh] ${
```

Comment:
```suggestion
                className={`flex flex-col h-[79vh] ${
```

Fix the whitespace here